_data/guides.yaml

```diff
@@ -28,6 +28,9 @@ categories:
       - title: Identity Propagation with OpenID Connect
         url: /guides/security-oidc-identity-propagation
         description: Learn how to propagate identities within a deployment and across deployments when securing apps with OpenID Connect.
+      - title: Securing a WildFly Application Using OpenID Connect With Additional Scope Values
+        url: /guides/elytron-oidc-client-scope
+        description: How to secure a WildFly application deployed to OpenShift with OpenID Connect using additional scope values.
```

Comment:
Learn how to secure applications deployed to WildFly on OpenShift with OpenID Connect using additional scope values.

Reply:
fixed!